.eslintrc.json

```diff
@@ -48,13 +48,7 @@
 			}
 		],
 		"no-throw-literal": "warn",
-		"semi": "off",
-		"no-restricted-properties": [
-			2,
-			{
-				"property": "activeTextEditor"
-			}
-		]
```

Comment:
Because now we are using `ide().activeTextEditor` all over the place. But as we talked about none of the Cursorless code is now excepting a pure vscode editors they can't really do anything with it. ie if you incorrectly fetch the wrong editor it will become very apparent

Reply:
https://github.com/cursorless-dev/cursorless/issues/1136